modules/gcp-gke/inputs.tf

```diff
@@ -178,3 +178,9 @@ variable "maintenance_policy_config" {
   description = "(OPTIONAL) A list of objects used to configure maintenance policy "
   default     = []
 }
+
+variable "nat_ip_addresses" {
```

Comment:
`nat_ip_addresses` should be declared as input variable in `terraform-gcp-gke` module too.

Reply:
done :)